Tmain/readtags-qualifier-op-downcase-upcase.d/run.sh

```diff
@@ -27,5 +27,17 @@ ${V} ${READTAGS} -t output.tags -Q '(member "a" (downcase $inherits))' -l
 echo '# (member "b" (downcase $inherits))'
 ${V} ${READTAGS} -t output.tags -Q '(member "b" (downcase $inherits))' -l
 
-echo '# (member "d" (downcase $inherits))'
+echo '# (member "c" (downcase $inherits))'
```

Comment:
This change is not needed. I specified "d" as an object that is not in `(downcase $inherits)`.
I expected no output for ths filter condition.